app/views/lists/_list.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container-fluid">
+	<div class="row">
+		<div class="col-md-12">
+			<nav class="navbar navbar-default" role="navigation">

```

Comment:
Let's talk tomorrow what you're going for with these `navbar` elements. Navbar is traditionally just the top of an html page (ie, just the persistent top navigation bar on a page), I can help refactor this.